### DISCUSSION TITLE
Tim are going to remove `target: 'serverless'` option in nextJS

### DISCUSSION THREAD

**Discussion by jinqian1998** [serverless-nextjs/serverless-next.js#2414] [General]
Hello guys,
I noticed vercel are going to remove the `target: 'serverless'` option in the future, in my understand, we are still using this option to generate a customize pack.
NextJS have a new feature called "output files trace", and a new option called `outputStandalone`. @iiroj ii have just create a PR using this option to compile the next, thanks his effort!
But for now, this option is still experimental. I don't know if vercel will remove it one day...
Any idea about deploying it in newer nextJS version?

**Comment by iiroj** [comment]
I managed to get a PoC working, and the PR reflects that.

Basically the change means that instead of the lambda handler importing a serverless page and running its `renderPage` method, it has to spin up a Next.js server and directly handle the request.

The PoC is still missing all the sls-next features from around it, as I haven't had any time to move forward with it... maybe I should now pick up the effort, unless someone wants to take over. 😄

**Reply by jinqian1998** [reply]
> 
Thank you very much @iiroj .
We may change the deployment method if nextJS remove `serverless` target permanently.
